src/components/Reusables/CompactDeletableListItem/CompactDeletableListItem.tsx

```diff
@@ -0,0 +1,65 @@
+import {
+  Button,
+  ButtonVariant,
+  ListItem,
+  ButtonColor,
+  SvgIcon,
+} from '@altinn/altinn-design-system';
+import { useTranslation } from 'react-i18next';
+
+import { ReactComponent as MinusCircle } from '@/assets/MinusCircle.svg';
+
+import classes from './CompactDeletableListItem.module.css';
+
+export interface DeletableListItemProps {
```

Comment:
Could it be problematic that these props have the same name as the other (non-compact) DeletableListItem when they don't contain the same props?
Maybe rename this one to CompactDeletableListItemProps?

Reply:
Yes, good catch